src/dispatch/participant/flows.py

```diff
@@ -12,10 +12,9 @@
     ParticipantRoleType,
     ParticipantRoleCreate,
 )
+from dispatch.participant import service as participant_service
```

Comment:
I had to import the service module this way to fix a circular import I stumbled upon. Not sure why it did not manifest before.